{
    "CVE-ID": "CVE-2008-5983",
    "NVD": {
        "Descriptions": "Untrusted search path vulnerability in the PySys_SetArgv API function in Python 2.6 and earlier, and possibly later versions, prepends an empty string to sys.path when the argv[0] argument does not contain a path separator, which might allow local users to execute arbitrary code via a Trojan horse Python file in the current working directory.",
        "Cvss2_BaseScore": 6.9,
        "Cvss3_BaseScore": 0,
        "Cvss2_VectorString": "AV:L/AC:M/Au:N/C:C/I:C/A:C",
        "Cvss3_VectorString": "",
        "CPEs": [
            {
                "URI": "cpe:/o:fedoraproject:fedora:13",
                "FormattedString": "cpe:2.3:o:fedoraproject:fedora:13:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/o:canonical:ubuntu_linux:8.04::~~-~~~",
                "FormattedString": "cpe:2.3:o:canonical:ubuntu_linux:8.04:*:*:*:-:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/o:canonical:ubuntu_linux:11.10",
                "FormattedString": "cpe:2.3:o:canonical:ubuntu_linux:11.10:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/o:canonical:ubuntu_linux:11.04",
                "FormattedString": "cpe:2.3:o:canonical:ubuntu_linux:11.04:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/o:canonical:ubuntu_linux:10.04::~~-~~~",
                "FormattedString": "cpe:2.3:o:canonical:ubuntu_linux:10.04:*:*:*:-:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python",
                "FormattedString": "cpe:2.3:a:python:python:*:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "2.6.6",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python",
                "FormattedString": "cpe:2.3:a:python:python:*:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "3.1.0",
                "VersionEndExcluding": "3.1.3",
                "VersionEndIncluding": ""
            }
        ]
    },
    "Official": {
        "2.7rc1": {
            "source": [
                "https://github.com/python/cpython/blob/v2.7.15rc1/Misc/NEWS.d/2.7rc1.rst",
                "https://github.com/python/cpython/blob/v2.7rc1/Misc/NEWS"
            ],
            "BTS": {
                "bpo5753": {
                    "ID": "bpo5753",
                    "url": "https://bugs.python.org/issue5753",
                    "date": "8070",
                    "nonce": "205374",
                    "section": "C API"
                }
            }
        },
        "2.6.6rc1": {
            "source": [
                "https://github.com/python/cpython/blob/2.6/Misc/NEWS"
            ],
            "BTS": {
                "bpo5753": {
                    "ID": "bpo5753",
                    "url": "https://bugs.python.org/issue5753",
                    "date": "",
                    "nonce": "",
                    "section": ""
                }
            }
        },
        "3.1.3rc1": {
            "source": [
                "https://github.com/python/cpython/blob/3.1/Misc/NEWS"
            ],
            "BTS": {
                "bpo5753": {
                    "ID": "bpo5753",
                    "url": "https://bugs.python.org/issue5753",
                    "date": "",
                    "nonce": "",
                    "section": ""
                }
            }
        },
        "3.2a1": {
            "source": [
                "https://github.com/python/cpython/blob/3.2/Misc/NEWS"
            ],
            "BTS": {
                "bpo5753": {
                    "ID": "bpo5753",
                    "url": "https://bugs.python.org/issue5753",
                    "date": "",
                    "nonce": "",
                    "section": ""
                }
            }
        }
    }
}